src/extensions/cp/apple/finalcutpro/inspector/audio/AudioConfiguration.lua

```diff
@@ -0,0 +1,137 @@
+--- === cp.apple.finalcutpro.inspector.audio.AudioConfiguration ===
+---
+--- The Audio Configuration section of the Audio Inspector.
+
+local require = require
+
+local log                               = require("hs.logger").new("audioConfiguration")
+
+local axutils                           = require("cp.ui.axutils")
+local Element                           = require("cp.ui.Element")
+local CheckBox                          = require("cp.ui.CheckBox")
+local just                              = require("cp.just")
+local MenuButton                        = require("cp.ui.MenuButton")
+
+local Do                                = require("cp.rx.go.Do")
+local If                                = require("cp.rx.go.If")
+local Throw                             = require("cp.rx.go.Throw")
+local Require                           = require("cp.rx.go.Require")
+
+local sort = table.sort
+
+--------------------------------------------------------------------------------
+--
+-- THE MODULE:
+--
+--------------------------------------------------------------------------------
+local AudioConfiguration = Element:subclass("AudioConfiguration")
+
+function AudioConfiguration.__tostring()
```

Comment:
This is unnecessary when setting the name to the full package, as suggested above.